node/image-classification-with-huggingface/src/setup.js

```diff
@@ -0,0 +1,96 @@
+import { Client, Databases, Storage } from "node-appwrite";
+import { throwIfMissing } from "./utils.js";
+import 'dotenv/config'
+
+console.log('Setting up database and storage');
+
+throwIfMissing(process.env, [
+  'APPWRITE_BUCKET_ID',
+  'APPWRITE_DATABASE_ID',
+  'APPWRITE_COLLECTION_ID',
+  'APPWRITE_API_KEY',
+  'APPWRITE_FUNCTION_PROJECT_ID'
+]);
+
+const client = new Client();
+client
+  .setEndpoint(
+    process.env.APPWRITE_ENDPOINT ?? 'https://cloud.appwrite.io/v1'
+  )
+  .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID)
+  .setKey(process.env.APPWRITE_API_KEY);
+
+setup(client);
+
+async function setup(client) {
+  const database = new Databases(client);
+  try {
+    await database.get(process.env.APPWRITE_DATABASE_ID);
+  } catch (err) {
+    if (err.code === 404) {
+      console.log('Creating database');
+      await database.create(
```

Comment:
Do we need to GET it first and then create? Can't we always just create first and ignore if the response is 409? 
We can shorten this code up a lot. The GET seems redundant